activestorage/lib/active_storage/service/azure_storage_service.rb

```diff
@@ -1,19 +1,18 @@
 # frozen_string_literal: true
 
 require "active_support/core_ext/numeric/bytes"
-require "azure/storage"
-require "azure/storage/core/auth/shared_access_signature"
+require "azure/storage/blob"
```

Comment:
Match the S3 and GCS services in ensuring a compatible client gem is activated:

```suggestion

gem "azure-storage-blob", "~> 1.1"
require "azure/storage/blob"
```

Reply:
Good suggestion! Thanks!